webapi/src/main/scala/org/knora/webapi/slice/v3/projects/api/model/ProjectsDto.scala

```diff
@@ -0,0 +1,166 @@
+/*
+ * Copyright © 2021 - 2025 Swiss National Data and Service Center for the Humanities and/or DaSCH Service Platform contributors.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.knora.webapi.slice.v3.projects.api.model
+
+import sttp.tapir.Codec
+import sttp.tapir.CodecFormat
+import sttp.tapir.Schema
+import sttp.tapir.Validator
+import zio.json.*
+
+import org.knora.webapi.slice.v3.projects.domain.model.*
+import org.knora.webapi.slice.v3.projects.domain.model.DomainTypes.*
+
+object ProjectsDto {
+
+  final case class ProjectShortcodeParam(value: String) {
```

Comment:
no need to duplicate this, you can use Shortcode directly, see other endpoints doing this.